doc/api/child_process.markdown

```diff
@@ -376,7 +84,8 @@ callback or returning an EventEmitter).
   * `stderr` {Buffer}
 * Return: ChildProcess object
 
-Runs a command in a shell and buffers the output.
+Spawns a shell then executes the `command` within that shell, buffering any
+generated output.
```

Comment:
The distinction between exec and execFile is just efficiency, pretty much, on Unix. On Windows, the distinction is MUCH more imporant: .bat and .cmd files are NOT executable. So for example, all npm installed node CLIs can NOT be execFile() or spawn() ed on Windows... they can only be run by a shell, since cmd.exe knows how to run them.

Its a huge portability pitfall, and the current docs gloss over that.